### PR TITLE
Minor updates to the release guides

### PR DESCRIPTION
**Pull request by ephraimbuddy**

### CODE REVIEW

File: dev/README_RELEASE_PYTHON_CLIENT.md [hunk 1]
```diff
@@ -561,10 +564,10 @@ done
 # Remove old release
 cd ..
 svn rm ${PREVIOUS_VERSION}
-svn commit -m "Release Apache Airflow Python Client ${VERSION} from ${RC}"
+svn commit -m "Release Apache Airflow Python Client ${VERSION} from ${VERSION}${VERSION_SUFFIX}"
 ```
 
-Verify that the packages appear in [airflow](https://downloads.apache.org/airflow/clients/python/)
+Verify that the packages appear in [airflow](https://dist.apache.org/repos/dist/release/airflow/clients/python)
```
Comment (ephraimbuddy) [inline dev/README_RELEASE_PYTHON_CLIENT.md:570]: The downloads link takes a little bit of time to update

File: dev/breeze/src/airflow_breeze/commands/minor_release_command.py [hunk 2]
```diff
@@ -69,7 +69,7 @@ def commit_changes(version_branch):
         if DRY_RUN:
             console_print("Skipping below command on CI")
         run_command(
-            ["git", "commit", "-m", f"Update default branches for {version_branch}"],
+            ["git", "commit", "-m", f"Update default branches for {version_branch}", "--no-verify"],
```
Comment (ephraimbuddy) [inline dev/breeze/src/airflow_breeze/commands/minor_release_command.py:72]: Allowing this to run `pre-commit` gives room to updating hashes which we can do in a separate commit

File: dev/README_RELEASE_AIRFLOW.md [hunk 3]
```diff
@@ -315,11 +315,11 @@ The Release Candidate artifacts we vote upon should be the exact ones we vote ag
 
 - Create issue in github for testing the release using this subject:
 
-  ```shell script
-  cat <<EOF
-  Status of testing of Apache Airflow {VERSION}
-  EOF
-  ```
```
Comment (kaxil) [inline dev/README_RELEASE_AIRFLOW.md]: nit: Any reason for this change?

This change before puts the code-block under the bullets with same indent

Comment (ephraimbuddy) [reply dev/README_RELEASE_AIRFLOW.md]: It wasn't looking good on my IDE. The 'cat' command was not coloured. I will revert this change since it moved out of the indentation

**Review by kaxil — APPROVED** [review]

File: dev/README_RELEASE_PYTHON_CLIENT.md (hunk 1 above)
Comment (potiuk) [reply dev/README_RELEASE_PYTHON_CLIENT.md:570]: > The downloads link takes a little bit of time to update

Strange - it should not be (any more). It used to be like that, before ASF Switched to use CDN some 1.5 years ago - since then I never saw a delay. Maybe it's just the matter of Shift-Refresh?

**Review by potiuk — APPROVED** [review]

File: dev/README_RELEASE_PYTHON_CLIENT.md (hunk 1 above)
Comment (potiuk) [reply dev/README_RELEASE_PYTHON_CLIENT.md:570]: But it does not matter. really :)

Comment (ephraimbuddy) [reply dev/README_RELEASE_PYTHON_CLIENT.md:570]: During the last Python client release, it took some minutes before reflecting while the dist was available immediately, and that helped in confirming that everything worked fine

Comment (potiuk) [reply dev/README_RELEASE_PYTHON_CLIENT.md:570]: I see . Interesting :)

**Review by hussein-awala — APPROVED** [review]
